src/Algebra.jl

```diff
@@ -83,8 +83,9 @@ function Algebra.allocate_vector(::Type{<:PVector{T,A}},ids::PRange) where {T,A}
   PVector(values,ids)
 end
 
-struct FullyAssembledRows end
-struct SubAssembledRows end
+abstract type DistributedAssemblyStrategy end
```

Comment:
A minor comment on this name, FYI ... (no action required, neither). 

In the current notation for `GridapDistributed`, without exception, we use the name `DistributedXXX` to refer to the counterpart of the Gridap type `XXX`. `DistributedXXX` is typically implemented as local portions + gluing info among portions. 

Here `DistributedAssemblyStratregy` is neither the counterpart of `AssemblyStrategy` nor wraps local portions + gluing info. 

In any case, I think the name is appropriate. This controls the algorithm that performs assembly of vectors and matrices in a distributed-memory environment.